components/SwipeRow.js

```diff
@@ -134,24 +140,35 @@ class SwipeRow extends Component {
 		}
 	}
 
-	handlePanResponderEnd(e, gestureState) {
-		// re-enable scrolling on listView parent
+	ensureScrollEnabled() {
 		if (!this.parentScrollEnabled) {
 			this.parentScrollEnabled = true;
 			this.props.setScrollEnabled && this.props.setScrollEnabled(true);
 		}
+	}
+
+	handlePanResponderEnd(e, gestureState) {
+
+		// decide how much the velocity will affect the final position that the list item settles in.
+		const swipeToOpenVelocityContribution = this.props.swipeToOpenVelocityContribution;
+		const possibleExtraPixels = this.props.rightOpenValue * (swipeToOpenVelocityContribution);
+		const clampedVelocity = gestureState.vx > 5 ? 5 : gestureState.vx
```

Comment:
How about:
```
const clampedVelocity = Math.min(gestureState.vx, 5);
```
Feels a little cleaner

(also throw a semicolon on the end of this line please)

Reply:
agree.